infra/storage/spanner/migrations/000009.sql

```diff
@@ -0,0 +1,51 @@
+-- Copyright 2024 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- SavedSearches contains the most current revisions of saved searches.
+--
+-- A future table will contain the historical revisions of saved searches
+-- allowing for auditing and reverting to older versions of the saved search.
+CREATE TABLE IF NOT EXISTS SavedSearches (
+    ID STRING(36) NOT NULL DEFAULT (GENERATE_UUID()),
+    -- Name is the name of the saved search.
+    Name STRING(MAX) NOT NULL,
+    -- Query is the query string of the saved search.
+    Query STRING(MAX) NOT NULL,
+    -- Scope is the scope of the saved search, which can be one of the following:
+    -- USER_PUBLIC: The saved search is created by a user and is publicly accessible.
```

Comment:
I assume that there are other potential enum values reserved for future use, but having just one possible value without any mention of others reads like something is missing.

Reply:
That's correct that there is only one possible value now.

In the future, there could be USER_PRIVATE (private ones) or GLOBAL (so bookmarks like the Baseline 2023 bookmark are not hardcoded in the frontend)

Reply:
I changed the comment to refer to source code for possible values.